packages/headless/src/app/reducers.ts

```diff
@@ -49,11 +47,13 @@ export const querySuggest = querySuggestReducer;
 export const sortCriteria = sortCriteriaReducer;
 export const context = contextReducer;
 export const didYouMean = didYouMeanReducer;
+export const resultPreview = resultPreviewReducer;
+
+export const debug = debugReducer;
+export const facetOrder = facetOrderReducer;
 export const fields = fieldsReducer;
 export const pipeline = pipelineReducer;
 export const searchHub = searchHubReducer;
-export const debug = debugReducer;
-export const resultPreview = resultPreviewReducer;
 export const version = versionReducer;
```

Comment:
I've grouped these reducers together since, except for `debug`, they are not added by any controller at the moment.

The `debug` is a special case too since if the reducer is not there, it will not be possible to toggle the value via the url.